machine-setup/Makefile

```diff
@@ -0,0 +1,7 @@
+target: flake8 docker-build
+
+flake8:
+	flake8 *.py
+
+docker-build:
+	docker build -t jimeng/longhorn-test .
```

Comment:
Don't point to your personal repo here. Just point to `rancher/longhorn-env-setup`. Upload can be done manually later.

Also I think `longhorn-env-setup` sounds better. Because it's not the only component in `longhorn-test`.